backend/routers/projects.router.js

```diff
@@ -1,4 +1,4 @@
-const express = require("express");
+const express = require('express');
```

Comment:
Can you please drop this change as per other comments in this PR review re: linting